### PR TITLE
User form for localStorage and personalized Greeting

### PR DESCRIPTION
**Pull request by czintel**
On the start page, I created a user form that transfers the user name into localStorage and via useState displays the entered name in a personalized greeting.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -14,33 +14,43 @@ function App() {
 
   return (
     <Router>
-      <Wrapper>
+      <AppGrid>
         {/* <ScrollToTop elementToScrollUp={mainRef} /> */}
```
Comment (macfrei) [inline src/App.js:18]: Please remove unused code!

File: src/App.js [hunk 2]
```diff
@@ -14,33 +14,43 @@ function App() {
...
         <Header>
           <Logo src={appLogo} alt="App Logo" />
         </Header>
+
         <Main ref={mainRef}>
```
Comment (macfrei) [inline src/App.js:23]: What do you need the ref for?

File: src/App.js [hunk 3]
```diff
@@ -14,33 +14,43 @@ function App() {
...
           <Switch>
             <Route exact path="/">
-              <HomePage onSubmit={userName => setUserName(userName)} />
```
Comment (macfrei) [inline src/App.js:25]: I like this short version better than extracting it to it's own function, but that's personal preference 🙂

File: src/pages/HomePage.js [hunk 4]
```diff
@@ -58,13 +61,35 @@ const PageWrapper = styled.section`
...
+
+  label {
+    display: grid;
+    gap: 4px;
+    font-weight: bold;
+  }
+
+  input {
+    padding: 4px;
+  }
```
Comment (macfrei) [inline src/pages/HomePage.js:77]: Could you not add these styles to your `LabeledInput` component?

File: src/pages/LearningPage.js [hunk 5]
```diff
@@ -3,25 +3,22 @@ import PropTypes from 'prop-types'
...
+LearningPage.propTypes = {
   userName: PropTypes.string,
 }
 
-export default function LearningCardList({ userName }) {
-  const nameOfUser = localStorage.getItem(userName)
-  console.log(nameOfUser)
-
+export default function LearningPage({ userName }) {
+  console.log(userName)
```
Comment (macfrei) [inline src/pages/LearningPage.js:11]:
```suggestion
```

**Review by macfrei — COMMENTED** [review]
Well done! 🚀

File: src/pages/HomePage.js (hunk 4 above)
Comment (czintel) [reply src/pages/HomePage.js:77]: I might at a later point. I'm sure I will use `LabeledInput` in other components later this week and want to keep the styling in HomePage until then.

**Pull request merged**